src/util/hd-shortcuts.c

```diff
@@ -0,0 +1,266 @@
+#include "hd-shortcuts.h"
+
+#include <glib.h>
+#include <glib/gprintf.h>
+#include <gtk/gtk.h>
+#include <gdk/gdkx.h>
+
+#include <time.h>
+#include <sys/time.h>
+
+#include <matchbox/core/mb-wm.h>
+#include <matchbox/mb-wm-config.h>
+#include <matchbox/core/mb-wm-object.h>
+#include <matchbox/comp-mgr/mb-wm-comp-mgr.h>
+
+#include "hildon-desktop.h"
+#include "hd-wm.h"
+#include "hd-theme.h"
+#include "hd-util.h"
+#include "hd-dbus.h"
+#include "hd-volume-profile.h"
+#include "../launcher/hd-app-mgr.h"
+#include "../home/hd-render-manager.h"
```

Comment:
I'd rather move that file in the upper directory, that way there will be no need for "../$whatever" includes. But no strong preference.

Reply:
> Seems there are mixed functional changes and moved code here, please either split code movement from new/changed functionality to separate commits or explain in details what has been changed (and how).

there is some moving of code, but mostly its a refactor, im not going back and splitting this pr like that, as due to the code structure its impractical to do so while ensuring that every commit compiles for future bisecting.